lib/thread_printer.rb

```diff
@@ -2,9 +2,17 @@
 
 module ThreadPrinter
```

Comment:
How can we find these logs in Mezmo? Should we print them using Clog?

Reply:
Mezmo has stdout and stderr. This is one of the things I like about it: as opposed to using a tool that insists on API requests from within the process, we can catch weird stuff, like stderr messages in libraries or ruby's core dump text. Sure, it's ugly (and so is this), but you can get at it.

 The general idea was this was a similar kind of dump we want to get no matter what, it's intended to be exceptional. That's also why I don't frame it as JSON (also because: I _have_ framed it with JSON in the past only to find, reasonably, very large JSON messages with full backtraces are rejected)